pkg/webhook/policy.go

```diff
@@ -419,9 +419,16 @@ func (h *validationHandler) validateTemplate(ctx context.Context, req *admission
 	return false, nil
 }
 
+func getAnyObject(req *admission.Request) []byte {
+	if req.AdmissionRequest.Operation == admissionv1.Delete {
+		return req.AdmissionRequest.OldObject.Raw
+	}
+	return req.AdmissionRequest.OldObject.Raw
```

Comment:
shouldn't it be `req.AdmissionRequest.Object.Raw` for non DELETE operation?
Both paths return the same result right now

Reply:
Yip! good catch. Pushing the change.